PermissionsEx/permissions.yml

```diff
@@ -1113,6 +1113,8 @@ groups:
         permissions:
         - ch.alias.mr.shifty
         - -essentials.protect.damage.fall
+        - worldedit.navigation.thru.command
+        - worldedit.navigation.jumpto.command 
```

Comment:
in this gif, he is seen teleporting where ever he sees : https://www.gamingonlinux.com/uploads/articles/article_images/1472649950Destruction.gif